ci/scripts/install-standalone-vm.bash

```diff
@@ -0,0 +1,441 @@
+#!/usr/bin/env bash
+
+# script that installs teet app, postgresql db, datomic local, postgrest api on one vm.
+#
+# usage:
+# run this from shell on your dev machine:
+# ci/scripts/install-standalone-vm.bash launchvm <ssh-key-name>
+# where ssh key name a key name from the registered ec2 key pairs in the aws env.
+# This will start the vm, detect what git branch you are on and try to check out that same branch at the remote vm.
+
+
+MACHINE_ID=b # pick one from a-e that doesn't conflict with someone else
+
+
+# exit on unhandled error status, error on undefined var references, consider pipelines failed when either side fails
+set -euo pipefail 
+
+TEET_GIT_URL=https://github.com/solita/mnt-teet.git
+export AWS_REGION=eu-central-1
+export AWS_DEFAULT_REGION=$AWS_REGION
+
+# used just for postgres password now
+function gensecret {
+    name="$1"
+    touch "$name.secret"
+    chmod 600 "$name.secret"
+    head -c 42 /dev/urandom | base64 > "$name.secret"
+}
+
+function patient-docker-pull {
+    for x in $(seq 10); do
+	if docker pull "$1"; then
+	    break
+	else
+	    echo docker pull failed, retrying after 30s sleep, retry $x of 10
+	    sleep 30
+	fi
+    done
+}
+
+
+function start-teet-app {
+    local DB_URI="postgres://authenticator@teetdb:5432/teet"
+    local JWT_SECRET="$(cat jwt.secret)"
+    mkdir -p mnt-teet-private
+
+    patient-docker-pull postgrest/postgrest
+    
+    docker run -d --network docker_teet --name teetapi -p 127.0.0.1:3000:3000 \
+       -e PGRST_DB_URI="$DB_URI" \
+       -e PGRST_DB_ANON_ROLE="teet_anon" \
+       -e PGRST_DB_SCHEMA="teet" \
+       -e PGRST_JWT_SECRET="$JWT_SECRET" \
+       postgrest/postgrest    
+    cat > ~/.datomic/dev-local.edn <<EOF
+{:storage-dir "/home/root/.datomic/dev-local-data"}
+EOF
+    
+    cat > mnt-teet-private/config.edn <<EOF
+{:datomic
+ {:db-name "teet"
+  :asset-db-name "asset"
+  :client {:server-type :dev-local
+           :storage-dir "$HOME/.datomic/dev-local-data"
+           :system "teet-test-env"}}
+ :listen-address "0.0.0.0"
+ :auth {:jwt-secret "$JWT_SECRET"
+        :basic-auth-password "$(ssm-get /teet/api/basic-auth-password)"}
+ :env :dev
+ :api-url "https://$MYDNS:3443"
+ :enabled-features #{:road-information-view
+                     :meetings
+                     :component-view
+                     :data-frisk
+                     :my-role-display
+                     :dummy-login
+                     :cooperation
+                     :admin-inspector
+                     ; :vektorio
+                     :asset-db
+                     :cost-items
+                     :land-owner-opinions
+                     :assetmanager}
```

Comment:
how do we remember to update the list of enabled features when we add new ones?

Reply:
Good question, I thought about a few different ways to solve it but didn't come up with a good idea yet.